### PR TITLE
Pagination sorting swagger implemented

### PR DESCRIPTION
**Pull request by RomanYuzviak**

### CODE REVIEW

**Review by don-bigdad — APPROVED** [review]
nice

File: src/main/java/com/example/onlinebookstore/controller/BookController.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+//swagger doc http://localhost:8080/swagger-ui/index.html#/
```
Comment (Elena-Bruyako) [inline src/main/java/com/example/onlinebookstore/controller/BookController.java]: Remove comment

File: src/main/java/com/example/onlinebookstore/controller/BookController.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+@Tag(name = "Book CRUD API", description = "Endpoints to manage books")
```
Comment (Elena-Bruyako) [inline src/main/java/com/example/onlinebookstore/controller/BookController.java]:
```suggestion
@Tag(name = "Book store API", description = "Endpoints to manage books")
```

File: src/main/java/com/example/onlinebookstore/controller/BookController.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+@RequestMapping(value = "/api/books")
+public class BookController {
+    private final BookService bookService;
+
+    @Autowired
+    public BookController(BookService bookService) {
+        this.bookService = bookService;
+    }
+
+    @GetMapping
```
Comment (Elena-Bruyako) [inline src/main/java/com/example/onlinebookstore/controller/BookController.java:31]: It's better to add @Operation for all endpoints

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Good job! 
There are conflicts in your branch, need to resolve them. 
Also, try to figure out how to merge your previous changes. Because in every your PRs I see the same code from previous PR with the same mistakes.

File: pom.xml [hunk 4]
```diff
@@ -61,6 +61,19 @@
             <groupId>org.liquibase</groupId>
             <artifactId>liquibase-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+        </dependency>
```
Comment (liliia-ponomarenko) [inline pom.xml:67]: Do you really need both annotations? you already have `spring-boot-starter-validation`

**Review by liliia-ponomarenko — APPROVED** [review]
Great 😊

**Pull request merged**